R/column_stats.R

```diff
@@ -1,3 +1,7 @@
+
+library(modeest)
+library(dplyr)
```

Comment:
These imports can be removed. 
```suggestion

```
We can add these as dependencies by running `use_package("package_name")`